data/data/libvirt/main.tf

```diff
@@ -30,11 +30,6 @@ resource "libvirt_ignition" "master" {
   content = "${var.ignition_master}"
 }
 
-resource "libvirt_ignition" "worker" {
-  name    = "${var.tectonic_cluster_name}-worker.ign"
```

Comment:
If we are transitioning (or already have transitioned?) to having the actuator create these for us, then we need to ensure they're creating cluster-name prefixed volumes so our destroy code picks them up (#660).  Have you checked that?